README.md

```diff
@@ -194,6 +194,28 @@ This will create a file `ccm-linode.yaml` which you can use to deploy the CCM.
 
 Note: Your kubelets, controller-manager, and apiserver must be started with `--cloud-provider=external` as noted in the following documentation.
 
+## Deployment Through Helm Chart
+
+Use the helm chart located under './deploy/chart'. This dir has the manifest for Linode CCM. There are two arguments required.
+
+The first argument must be a Linode APIv4 Personal Access Token with all permissions.
+(https://cloud.linode.com/profile/tokens)
+
+The second argument must be a Linode region.
+(https://api.linode.com/v4/regions)
```

Comment:
I think this should be a hyperlink on [region]

Reply:
Is there a page where all the regions are listed? I was only able to find this link to get all the regions available. But that link also gives a whole lot of other info